imgui.go

```diff
@@ -393,6 +393,21 @@ func SliderFloat(label string, value *float32, min, max float32) bool {
 	return SliderFloatV(label, value, min, max, "%.3f", 1.0)
 }
 
+// SliderFloat3V creates slider for a 3D vector
+func SliderFloat3V(label string, values *[3]float32, min, max float32, format string, power float32) bool {
+	labelArg, labelFin := wrapString(label)
+	defer labelFin()
+	formatArg, formatFin := wrapString(format)
+	defer formatFin()
+	cvalues := (*C.float)(&values[0])
+	return C.iggSliderFloatN(labelArg, cvalues, 3, C.float(min), C.float(max), formatArg, C.float(power)) != 0
+}
+
+// SliderFloat3 creates slider for a 3D vector
```

Comment:
Please have it state `SliderFloat3 calls SliderFloat3(label, values, min, max, "%.3f", 1.0).` to indicate the used "defaults".